src-ts/lib/loading-spinner/LoadingSpinner.module.scss

```diff
@@ -0,0 +1,24 @@
+@import '../styles';
+
+.loading-spinner {
+    position: fixed;
```

Comment:
let's change this so that it fills the content section of the page only, centered horizontally and vertically.

so we can see the topnav and page header